modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Numeric/Numeric.es.js

```diff
@@ -243,7 +243,6 @@ const Main = ({
 				}}
 				onFocus={onFocus}
 				placeholder={placeholder}
-				predefinedValue={predefinedValue}
```

Comment:
@linolaoj, why did you decide to remove it? Can you please clarify?

Reply:
Hi @natocesarrego , this _predefinedValue_ parameter was being passed to _Numeric but_ inside _Numeric_ it was never used, and it was also passed to _ClayInput_ inside _otherProps_ causing the test to fail because _ClayInput_ doesn't recognize _predefinedValue_ parameter.

Reply:
Thanks for your explanation, @linolaoj. :wink: